tests/cli/workflow/rnafusion/test_cli_rnafusion_run.py

```diff
@@ -165,11 +166,88 @@ def test_with_revision(
 
     # WHEN dry running with dry specified
     result = cli_runner.invoke(
-        run, [case_id, "--dry-run", "--revision", "2.1.0"], obj=rnafusion_context
+        run, [case_id, "--dry-run", "--from_start", "--revision", "2.1.0"], obj=rnafusion_context
```

Comment:
```suggestion
        run, [case_id, "--dry-run", "--from-start", "--revision", "2.1.0"], obj=rnafusion_context
```
In the CLI you usually use `-` as the separator (kebab case?) which is a different convention to the snake case (and camelCase for objects) in python.